texar/torch/data/tokenizers/bert_tokenizer.py

```diff
@@ -74,6 +74,10 @@ class BERTTokenizer(PretrainedBERTMixin, TokenizerBase):
         'scibert-scivocab-cased': 512,
         'scibert-basevocab-uncased': 512,
         'scibert-basevocab-cased': 512,
+
+        # BERT for MS-MARCO
+        'bert-msmarco-base': 512,
+        'bert-msmarco-large': 512,
```

Comment:
This won't be the last/best Bert model for MS-Marco, so probably we'd come up with more specific names, say bert-msmarco-nguyen2019

Reply:
Sure let me change that

Reply:
Changed the names